test/benchgc/src/benchgc/__main__.py

```diff
@@ -124,6 +124,20 @@ def add_common_options(parser: argparse.ArgumentParser):
         help="if we need print the ir during the pass-pipeline",
     )
 
+    parser.add_argument(
+        "--cpu_cache_sizes",
+        required=False,
+        help="set the cpu cache sizes, format: L1:L2:L3",
```

Comment:
Can you provide a detailed example for better understanding `set the cpu cache sizes in bytes, format: L1:L2:L3 (e.g. 32768:1048576:33554432)`.

Reply:
The Readme was updated for the new args.
### --cpu_cache_sizes, --max_vector_width
* BenchGC will automatically obtain target info and add the DLTI attr to the IR
* In some cases, if the system info obtained by BenchGC is not accurate, you can specify the relevant attributes for BenchGC through these options.
* --cpu_cache_sizes: cpu cache sizes in bytes, format: L1:L2:L3, example: `--cpu_cache_sizes 49152:2097152:110100480`
* --max_vector_width: the maximum width of vector registers available in a CPU, example `--max_vector_width `